client/app/components/composites/ManageAvailabilityHeader/ManageAvailabilityHeader.css

```diff
@@ -0,0 +1,35 @@
+.root {
+  position: relative;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  overflow: hidden;
+}
+
+.colorLayer {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  opacity: 0.9;
+  filter: blur(5px);
```

Comment:
I think the blur filter might be in the wrong layer. I guess it should blur the image, but now the image is sharp and clear but the color layer has some blurred white borders. See screenshot:

![screen shot 2016-11-25 at 16 05 22](https://cloud.githubusercontent.com/assets/429876/20627486/ff76aa52-b328-11e6-80ec-71f8f716abdd.png)